core/src/main/java/io/fabric8/maven/core/openshift/BuildService.java

```diff
@@ -63,7 +63,7 @@ public Build startBuild(OpenShiftClient client, File dockerTar, String buildName
             }
             if (exp.getCause() instanceof IOException && exp.getCause().getMessage().contains("Stream Closed")) {
                 log.error("Build for %s failed: %s", buildName, exp.getCause().getMessage());
-                log.error("If you are refering to an ImageStream as S2I builder image, please ensure that this ImageStream exists (with 'oc get is')");
+                log.error("If you are refering to an ImageStream as S2I builder image, please ensure that this ImageStream exists (with 'oc get is -n openshift')");
```

Comment:
From https://github.com/nicolaferraro/fabric8-maven-plugin/blob/2bd0d1ec0c39651d3653bd0269e87ffa724a6cab/core/src/main/java/io/fabric8/maven/core/openshift/BuildService.java#L56, we should be able to validate the IS for the user perhaps? Not sure, but we should at least be able to get the reference to the IS & hence the correct namespace for it.